DocumentedExamples/Zonally_Averaged_Global_Meridional_Overturning_Circulation.ipynb

```diff
@@ -6,9 +6,9 @@
    "source": [
```

Comment:
<p>I think we're unsure about this approach, maybe we could split this into two cells which are collapsed by default, and end up giving us the same inputs that we can then flow through the rest of the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
I tweaked this, but not sure if it's clear enough yet?